libs/SmartStore/SmartStore/Classes/SFSmartStore.h

```diff
@@ -163,43 +163,43 @@ extern NSString *const EXPLAIN_ROWS;
  @param storeName The name of the store.  If in doubt, use kDefaultSmartStoreName.
  @return A shared instance of a store with the given name.
  */
-+ (nullable id)sharedStoreWithName:(NSString*)storeName;
++ (nullable id)sharedStoreWithName:(NSString*)storeName NS_SWIFT_NAME(sharedStore(storeName:));
```

Comment:
@wmathurin Repetition of Store in the api. How about NS_SWIFT_NAME(sharedStore(name:)). The same for the rest of the api(s). Additionally could we change the signature to return an instancetype?